queries.md

```diff
@@ -1468,6 +1469,121 @@ DB::transaction(function () {
 });
 ```
 
+<a name="scopes"></a>
+## Scopes
```

Comment:
I wonder if that is "officially" considered a "scope" in the stricter sense 🤔

Reply:
Turns out: it isn't!